src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -212,6 +215,7 @@ export const SimplePowerBIReportEmbed = ({
           downloadLabel={labels.downloadButton}
         />
       )}
+      {hasUnknownStatus && <DashboardPlaceholder label={labels.hasUnknownStatus.label} />}
```

Comment:
This whole part (L198-L218) becomes hard to read and maintain, it's difficult to make sure that the boolean variables won't lead to several placeholders being displayed (that's probably why we missed the "unknown" status bug :sweat_smile:)
This can be done later though, I'll add a task somewhere to refactor this part :slightly_smiling_face: